modules/core/src/main/java/org/apache/ignite/internal/processors/query/GridRunningQueryInfo.java

```diff
@@ -155,6 +155,16 @@ public boolean cancelable() {
         return cancel != null;
     }
 
+    /**
+     * @return if query have been canceled. Always returns {@code false} for uncancellable queries.
+     */
+    public boolean isCanceled() {
+        if (cancel == null)
```

Comment:
Can be simplified: return cancel != null && cancel.isCanceled()

Reply:
Ok, done.